js/modules/sunburst.src.js

```diff
@@ -941,6 +942,18 @@ var sunburstSeries = {
         this.setShapeArgs(nodeTop, values, mapOptionsToLevel);
         // Set mapOptionsToLevel on series for use in drawPoints.
         series.mapOptionsToLevel = mapOptionsToLevel;
+
+        // #10669 - verify if all nodes have unique ids
+        series.data.forEach(function (child) {
+            if (nodeIds.indexOf(child.id) > -1) {
```

Comment:
Would it be faster or maybe simpler to use a map instead of the array and indexOf? E.g:
```javascript
if (nodeIds[child.id]) {
// throw error
}
nodeIds[child.id] = true;
```